option optimize_for = LITE_RUNTIME;

message LegionSkillOperateRequest
{
 required uint32 skill_id = 1;
}
